projects/packages/forms/.gitattributes

```diff
@@ -6,8 +6,9 @@ package.json      export-ignore
 # Files to include in the mirror repo, but excluded via gitignore
 # Remember to end all directories with `/**` to properly tag every file.
 # /src/js/example.min.js  production-include
-/src/**/*.min.css production-include
-/src/**/*.rtl.css production-include
+/src/**/*.min.css      production-include
+/src/**/*.rtl.css      production-include
+/src/dashboard/dist/** production-include
```

Comment:
I'm thinking it may be easier if that bundle lived at the root of the package, instead of in a subdirectory? What do you think?

Reply:
Kuba and I had this conversation yesterday and agreed to have it at the root since https://github.com/Automattic/jetpack/pull/28630 will also need a `dist` folder.

Reply:
Alright. I've updated it in a5147d31bbb4f3a17651c804bb24bee43282f920